core-skills/frontend-web-development/README.md

```diff
@@ -0,0 +1,107 @@
+- [Responsive Design](./resources/responsive-design.md)
+- [CSS Architecture](./resources/css-architecture.md)
+- [Web Typography](./resources/web-typography.md)
+- [Icons](./resources/icons.md)
+- [Cross Browser Testing](./resources/cross-browser-testing.md)
+- [Web Accessibility](./resources/web-accessibility.md)
+- [HTML/CSS Standards](./resources/html-css-standards.md)
+- [Progressive Enhancement](./resources/progressive-enhancement.md)
+- [Web Performance](./resources/web-performance.md)
+- [SEO](./resources/seo.md)
+
+## Assessment Levels
+
+For every level, as we should be "production ready" at every stage a candidate is expect to deliver:
+
+- Responsive "mobile first" HTML/CSS (breakpoints at 320px, 768px, & 1024px)
+- Pixel perfect components in their main browser at defined breakpoints, however when resizing the built html should reflow.
+- Cross Browser tested
+- Semantic HTML
+- Maintainable well architected CSS
+
+**Javascript is not required to complete any level.**
+
+### Rhino
+
+To obtain rhino a candidate has to present the following to the examiner:
+
+- A grid
+- A sticky header (remembering we're not testing JS)
+- Footer
+- Creation of any additional assets
+
+While presenting their pages, candidates should explain the following:
+
+- What image format did you pick for Logo
+- What are the different values of `position` and what do they do.
+- What are the different values of `display` and what do they do.
+- What tools if any did you use to make the icons?
+- Their initial CSS structure, and how they plan to keep it maintainable?
+- Their choice to use (or not use) an off the shelf grid?
+
+### Narwhal
+
+To obtain Narwhal a candidate has to present the following to the examiner:
+
+- Related Products
+- Promotions area
+- Main product image, and thumbnails
+- SEO
+
+While presenting their pages, candidates should explain the following:
+
+- Did you have to DRY up any CSS as you found common patterns?
+- What additional SEO features have you added? (HINT: Microformats)?
+- How would open graph meta tags help discoverability?
+- Which image format have you chosen to use? How would you optimize them if you had to?
+- The client wants a new promotion on the page and the design shows white text on a very light background. What would you recommend in this situation? If anything?
+
+### Unicorn
+
+To obtain unicorn a candidate has to present the following to the examiner:
+
+- Cart form
+- Accessible
+- Page load optimisations
+
+While presenting their pages, candidates should explain the following:
+
+- Did you have to change any of your previous HTML to follow accessibility guidelines?
+- What techniques did you use to speed up the page load?
+- How did you keep your form accessible whilst applying custom styling?
+- If you had to stop flash of unstyled content (FOUC) how would you have gone about it?
+- If a client requested the page structure to remain consistant while the page loaded how would you do that?
```

Comment:
Don’t fully understand this question. Does it imply it won’t? What do you mean by page structure?

Reply:
Perhaps there should be something in here about demonstrating simulating a slow connection ? And the impact on UX?

Reply:
Btw just the worst on news sites that break scroll position cos of adverts that pop out height